src/microbe_stage/components/StrainAffected.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+/// <summary>
+///   Contains variables related to strain
+/// </summary>
+[JSONDynamicTypeAllowed]
+public struct StrainAffected
+{
+    /// <summary>
+    ///   The current amount of strain
+    /// </summary>
+    public float CurrentStrain;
+
+    /// <summary>
+    ///   The amount of time the player has to wait before <see cref="CurrentStrain"/> sarts to fall
```

Comment:
This shouldn't explicitly talk about the player as other cells when the AI is updated should be able to sprint

Reply:
Should AI be updated in seperate PR to keep things clear or pack it here for consistency?

Reply:
I've let a bunch of features through that are missing the AI component, so I will let this PR go as well even without AI updates.